install/tools/man/ipa-adtrust-install.1

```diff
@@ -26,7 +26,7 @@ Adds all necessary objects and configuration to allow an IPA server to create a
 trust to an Active Directory domain. This requires that the IPA server is
 already installed and configured.
 
-Please note you will not be able to estabilish an trust to an Active Directory
+Please note you will not be able to establish an trust to an Active Directory
```

Comment:
This change is done in a different PR already: https://github.com/freeipa/freeipa/pull/714